sky/provision/gcp/config.py

```diff
@@ -242,24 +243,43 @@ def _is_permission_satisfied(service_account, crm, iam, required_permissions,
         # roles, so only call setIamPolicy if needed.
         return True, policy
 
-    for binding in original_policy['bindings']:
-        if member_id in binding['members']:
-            role = binding['role']
-            try:
-                role_definition = iam.projects().roles().get(
-                    name=role).execute()
-            except TypeError as e:
-                if 'does not match the pattern' in str(e):
-                    logger.info('_configure_iam_role: fail to check permission '
-                                f'for built-in role {role}. skipped.')
-                    permissions = []
+    logger.debug(f'_configure_iam_role: policy {json.dumps(original_policy, indent=2)}...')
```

Comment:
This seems to throw a lot of info. Shall we omit it for brevity/security?

Reply:
Good point! Removed. Thanks!